doc/source/io.rst

```diff
@@ -75,120 +75,200 @@ The two workhorse functions for reading text files (a.k.a. flat files) are
 :func:`~pandas.io.parsers.read_csv` and :func:`~pandas.io.parsers.read_table`.
 They both use the same parsing code to intelligently convert tabular
 data into a DataFrame object. See the :ref:`cookbook<cookbook.csv>`
-for some advanced strategies
+for some advanced strategies.
+
+Parsing options
+'''''''''''''''
+
+:func:`~pandas.io.parsers.read_csv` and :func:`~pandas.io.parsers.read_table`
```

Comment:
Can you use the top-level namespace here? So just ``` :func:`~pandas.read_csv` ``` (actually `` :func:`read_csv` `` will work as well and even cleaner)